M2/Macaulay2/e/CMakeLists.txt

```diff
@@ -199,7 +214,6 @@ set(SRCLIST
   poly
   polyquotient
   polyring
-  polyroots
```

Comment:
@mahrud I'm not completely sure that polyroots should be completely in "interface".

Reply:
Maybe...  But we might want to be able to call such a function in the engine.

Reply:
We can call interface functions in the engine. A number of engine files include interface files. Do you want to make some of them not available through the interface?